### PR TITLE
fix: Ghseets scroll

### LINKED ISSUE #6378: [Bug] Three scroll options are present on GSheets datasource query page
## Description
User is presented with 3 scroll options in a single space to scroll while querying on a datasource

### Steps
1. Create a Google Sheet datasource and choose to create a new query
2. Observe the scroll options on the right. There are 3 options presented to the user

Observed with Mongo query as well - but user is given 2 scroll options here.

### Important Details
- Version: Cloud
- OS: Win10
- Browser: Chrome
- Environment" Release

https://www.loom.com/share/e8cd51bf062d4a39850a9ff538e31a91

### PR DESCRIPTION
**Pull request by arunvjn**
## Description
Remove multiple scroll bars in GSheets.

Fixes #6378 

## Type of change
- Bug fix (non-breaking change which fixes an issue)

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes

## Test coverage results :test_tube:
<details><summary>:red_circle: Total coverage has decreased</summary>

    // Code coverage diff between base branch:release and head branch: fix/gsheet-scroll-bug 
Status | File | % Stmts | % Branch | % Funcs | % Lines 
 -----|-----|---------|----------|---------|------ 
 :red_circle: | total | 54.94 **(0)** | 36.82 **(-0.01)** | 33.63 **(0)** | 55.48 **(-0.01)**
 :red_circle: | app/client/src/utils/autocomplete/TernServer.ts | 50.47 **(-0.24)** | 37.72 **(-0.88)** | 36.21 **(0)** | 55.08 **(-0.27)**</details>

### CODE REVIEW

**Review by hetunandu — APPROVED** [review]

**Review by hetunandu — APPROVED** [review]

**Review by mohanarpit — APPROVED** [review]

**Review by hetunandu — APPROVED** [review]

File: app/client/cypress/support/commands.js [hunk 1]
```diff
@@ -664,8 +664,11 @@ Cypress.Commands.add(
     }
     cy.get(".string-value").contains(baseurl.concat(path));
     cy.get(".string-value").contains(verb);
-    cy.xpath(apiwidget.Responsetab)
-      .should("be.visible")
+    //cy.xpath(apiwidget.Responsetab)
```
Comment (mohanarpit) [inline app/client/cypress/support/commands.js:667]: Nit: Can we remove commented code if it's not required?

Comment (arunvjn) [reply app/client/cypress/support/commands.js]: Done

**Review by mohanarpit — APPROVED** [review]

**Pull request merged**